manager/test/api/src/test/java/io/apiman/manager/test/es/EsMetricsAccessorTest.java

```diff
@@ -88,7 +89,18 @@ private static RestHighLevelClient createEsClient() {
         config.put("client.host", node.getContainerIpAddress());
         config.put("client.port", node.getFirstMappedPort().toString());
         config.put("client.initialize", "true");
-        return new DefaultEsClientFactory().createClient(config, APIMAN_METRICS_INDEX_NAME, getDefaultIndices());
+        config.put("client.type", "es");
+
+        // We want the metrics client to initialise its indexes and this is an easy way of doing that.
```

Comment:
Please explain this :)

Reply:
The MetricsAccessorTest doesn't ever actually use the Metrics component to store its information. It uses a bulk upload method directly into the database. This means that the schema may not have been initialised.

To ensure that everything is done correctly, we start up the ESMetrics component, which will initialise its own schema.

If you look in the ESMetrics implementation you will see the corresponding (tiny) change that just ensure the client is initialised before it blocks.